pyproject.toml

```diff
@@ -166,3 +166,36 @@ messages_control.disable = [
 
 [tool.codespell]
 ignore-words-list = "hist, manuel, inactivate"
+
+[tool.pixi.workspace]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+# FIXME: remove python deps when https://github.com/prefix-dev/pixi/issues/532 is closed
+[tool.pixi.dependencies]
+# build
+cmake = "*"
+make = "*"
+
+# run
+geant4 = ">=11.2.2"
+clhep = "*"
+xerces-c = "*"
+hdf5 = "*"
```

Comment:
@ManuelHu this seems to install a non-threadsafe hdf5 version... do you understand why?